src/Symfony/Component/Cache/DependencyInjection/CacheCollectorPass.php

```diff
@@ -26,20 +26,9 @@
  */
 class CacheCollectorPass implements CompilerPassInterface
 {
-    private $dataCollectorCacheId;
-    private $cachePoolTag;
-    private $cachePoolRecorderInnerSuffix;
-
-    public function __construct(string $dataCollectorCacheId = 'data_collector.cache', string $cachePoolTag = 'cache.pool', string $cachePoolRecorderInnerSuffix = '.recorder_inner')
-    {
-        if (0 < \func_num_args()) {
-            trigger_deprecation('symfony/cache', '5.3', 'Configuring "%s" is deprecated.', __CLASS__);
-        }
-
-        $this->dataCollectorCacheId = $dataCollectorCacheId;
-        $this->cachePoolTag = $cachePoolTag;
-        $this->cachePoolRecorderInnerSuffix = $cachePoolRecorderInnerSuffix;
-    }
+    private string $dataCollectorCacheId = 'data_collector.cache';
+    private string $cachePoolTag = 'cache.pool';
+    private string $cachePoolRecorderInnerSuffix = '.recorder_inner';
```

Comment:
the values should be inlined instead

Reply:
Some values were multiple in a class. To prevent duplicated strings I used `private const` for them. Hope that is okay